modules/veteran_verification/app/serializers/veteran_verification/disability_rating_serializer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module VeteranVerification
+  class DisabilityRatingSerializer < ActiveModel::Serializer
+    attributes :rating_percentage, :effective_date, :decision
```

Comment:
There may be a semantic mismatch here - this looks like the effective date of the disability, from talking to OPM I thought they wanted the date of the rating? Or maybe that's a moot point if they refresh data from the API frequently enough that becomes the date of the rating? May be a point to circle back with OPM about.

Reply:
Here's what I read from the VA when deciding to use the effective date: https://www.benefits.va.gov/compensation/effective_dates.asp

From what I understood that date was what used when the VA paid benefits, so it made sense to me that that would be the date of the rating others would use. As far as I can tell the only other date we get from EVSS is a `related_disability_date` which seems not what OPM would need.

Reply:
Ok, I think this is right. One of OPMs concerns was that they don't want a veteran submit a very old disability letter for a temporary disability that is now resolved. But I think we avoid that by them refreshing data from the API endpoint on whatever frequency is sufficient for them. I.e that would be the equivalent of a veteran getting a new letter from VA.